src/components/SøknadsSteg/1-Søknadstype/Søknadstype.tsx

```diff
@@ -26,7 +26,7 @@ const Søknadstype: React.FC = () => {
                 }
                 defaultValue={søknad.søknadstype.verdi}
             >
-                <option value={ESøknadstype.IKKE_SATT}>{ESøknadstype.IKKE_SATT}</option>
+                <option value={ESøknadstype.IKKE_SATT}>Velg søknadstype</option>
                 <option value={ESøknadstype.ORDINÆR}>{ESøknadstype.ORDINÆR}</option>
                 <option value={ESøknadstype.UTVIDET}>{ESøknadstype.UTVIDET}</option>
                 <option value={ESøknadstype.EØS}>{ESøknadstype.EØS}</option>
```

Comment:
Har dere testet dette med backend? Der brukes hardkodede enums til å tolke søkandstypen. Så hvis verdien som sendes inn endres så vil det muligens føre til kræsj

Reply:
Ja vi sjekket det opp 😄 Det er value som blir sendt videre, det som står mellom option-tagsene er bare det som blir vist i browseren, så det burde funke sånn som det står nå

Reply:
Det kræsjer hos meg

Reply:
Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `main.kotlin.no.nav.familie.ba.Søknadstype` from String "Ordinær": not one of the values accepted for Enum class: [EØS, UTVIDET, ORDINÆR, IKKE_SATT]
 at [Source: (PushbackInputStream); line: 1, column: 54] (through reference chain: main.kotlin.no.nav.familie.ba.søknad.Søknad["søknadstype"]->main.kotlin.no.nav.familie.ba.søknad.Søknadsfelt["verdi"...

Reply:
ORDINÆR != Ordinær  
Feilen opptso teknisk sett ikke i denne PRen da, men i den forrige når enumene ble endret fra all caps.

Reply:
Det blir laget en mapper 👍